pkg/security/tests/process_test.go

```diff
@@ -139,11 +144,11 @@ func TestProcessContext(t *testing.T) {
 		},
 		{
 			ID:         "test_rule_ancestors_glob",
-			Expression: `exec.file.name == "ls" && exec.argv == "glob" && process.ancestors.file.path =~ "/usr/**"`,
+			Expression: `exec.file.name == "ls" && exec.argv == "glob" && process.ancestors.file.path in [~"/home/**", ~"/usr/**"]`,
```

Comment:
wondering why you need to add "home" here ?

Reply:
Yep .. It's a bit hacky, I will try to fix more properly this test.
The issue is that assumes that, `testsuite` have at least one ancestors present in /usr/** (like bash?). But with ebpfless, testsuite comes as the top lineage process (so, as it stands in ~/go/src/github[...]/tests/, I've added `/home/**` but definitively it's not rock solid.

Reply:
I've reworked the test on my last PR, using syscall tester as a proxy to execute ls, and we are now searching into `/tmp/***` instead (used to write the syscall tester on disk before executing it).